### ISSUE TITLE
Failed to update pie chart if it created with all values are 0 after v7.2.0

### ISSUE THREAD

**Issue by YuerLee** [ag-grid/ag-grid#6463]
**I'm submitting a ...**
```
[x] bug report => see 'Providing a Reproducible Scenario'
[ ] feature request => do not use Github for feature requests, see 'Customers of AG Grid'
[ ] support request => see 'Requesting Community Support'
```

**Current behavior**
In AG Charts v7.2.0, If create a pie chart with like `{a: 0, b:0}` and then update it with `{a:50, b:50}`, it would fail with error:

``` 
Charts - update error TypeError: Cannot set properties of undefined (setting 'datum')
```

Reproduced in v7.2.0 vs v7.1.0: https://stackblitz.com/edit/web-platform-kkh1kk?devToolsHeight=33&file=main.js

**Expected behavior**
Can update the chart.

**Please tell us about your environment:**
- AG Charts version: 7.2.0
- Browser: 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/112.0.0.0 Safari/537.36'
- Language: ES6+

**Comment by alexanderby** [comment]
Hi @YuerLee, the issue [is already fixed](https://github.com/ag-grid/ag-grid/pull/6408/commits/85aac350af759ab9a9ae8b55f7c358d0e0f00369) and it should work well in the next release.

**Comment by YuerLee** [comment]
Fixed in v29.3.0.
